TLS doesn't have to pad (#4821)
quicwg/base-drafts: draft-ietf-quic-tls.md

packet.

The TLS implementation does not need to ensure that the ClientHello is large

> enough that the Initial packet fills a datagram of the minimum size required for
> carring Initial packets. QUIC implementations use PADDING frames or packet
> coalescing to ensure that datagrams are large enough; see {{Section 14.1 of
> QUIC-TRANSPORT}}.

Reviewer: ```suggestion
enough to meet QUIC's requirements for Initial packets; see {{Section 14.1 of
QUIC-TRANSPORT}}. QUIC implementations use PADDING frames or packet coalescing
to ensure that datagrams are large enough.
```

Author: That is where I started, but we don't have requirements for packets, we have requirements for datagrams.  Maybe "...for datagrams that carry Initial packets..."

Reviewer: Ah, yes, makes sense. "datagrams that carry Initial packets" works.
